src/Servers/Connections.Abstractions/src/IConnectionFactory.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Connections
+{
+    /// <summary>
+    /// A factory abstraction for creating connections to an endpoint.
```

Comment:
The wording is funky here. Add these to other places too.

A factory abstraction for creating connections to an endpoint.
```suggestion
    /// A factory abstraction for creating connections for an endpoint.
```